olp-cpp-sdk-dataservice-write/test/unit/TestStreamLayerClient.cpp

```diff
@@ -2338,4 +2401,5 @@ TEST_P(StreamLayerClientCacheMockTest, FlushSettingsMaximumRequests) {
   flush_settings_.maximum_requests = 0;
   client_ = CreateStreamLayerClient();
   ASSERT_NO_FATAL_FAILURE(maximumRequestsSuccessAssertions(0, 10));
+  testing::Mock::VerifyAndClearExpectations(network_.get());
```

Comment:
Just interested - will be all expectation cleared if we will not reach this line (i.e. if the test failed at previous (2403) line)?